recipes/jina/meta.yaml

```
@@ -0,0 +1,130 @@
+{% set name = "jina" %}
+{% set version = "2.0.19" %}
+
+package:
+  name: {{ name|lower }}
```

The name of the package itself and one of the output should be different.
```suggestion
  name: {{ name|lower }}-split
```